packages/casting/test/fake-rpc-server.js

```diff
@@ -64,10 +64,19 @@ const fakeStatusResult = {
   },
 };
 
-export const startFakeServer = (t, fakeValues, marshaller = makeMarshal()) => {
+/** @typedef {Partial<import('ava').ExecutionContext<{cleanups: Array<() => void>}>> & {context}} fakeServerTestContext */
+/**
+ * @param {fakeServerTestContext} t
+ * @param {Array<{any}>} fakeValues
```

Comment:
What does `{any}` mean?

Reply:
IIUC correctly, it imposes no constraints beyond existence.